cmake/api_macros.cmake

```diff
@@ -44,6 +46,10 @@ macro(OPENDDS_GET_SOURCES_AND_OPTIONS
   set(${tao_options} ${_arg_TAO_IDL_OPTIONS})
   set(${opendds_options} ${_arg_OPENDDS_IDL_OPTIONS})
 
+  if (${_arg_SUPPRESS_ANYS} MATCHES ON OR ${_arg_SUPPRESS_ANYS} MATCHES OFF)
```

Comment:
DEFINED?

Reply:
I don't think that DEFINED works.  It also appears that what I have currently does not work on certain builds. I'll try to a find a solution.